middlewares/getUser.js

```diff
@@ -1,7 +1,13 @@
 import db from '../models';
 
 const getUser = (req, res, next) => {
-  db.User.findById(req.userId)
+  let whereField = {};
+  if (req.params.username) {
```

Comment:
Good job, this function really needs to be improved, thanks for taking initiative to do it. What if we completely move this function to the utilities class instead of leaving it as a standalone function?

Reply:
Moreover the inputs are gotten from the sent request, Can you please color, on why it's better off a Utility function?@Oloyedesinmiloluwa

Reply:
@Oloyedesinmiloluwa I don't think the `returnUser` in the `helpers` folder is needed anymore. I don't know I don't know if @veeqtor still needs it though. The `getUser` middleware already does the same thing.

Reply:
Nah,I don't need it.. the middleware is already doing the Job.. Guess I will clean it up. Thanks @Oloyedesinmiloluwa  for spotting this out..